client/src/components/CreateEvent.tsx

```diff
@@ -37,26 +37,25 @@ export const CreateEvent = () => {
     {
       createEvent: MutationResponse;
     },
-    InputEvent
+    { inputEvent: InputEvent }
   >(CREATE_EVENT_MUTATION);
 
   function handleEventCreation() {
     const inputEvent: InputEvent = {
       name: eventNameRef.current?.value || "",
       location: eventLocationRef.current?.value || "",
-      timeAndDate: eventTimeAndDateRef.current?.value || "",
+      timeAndDate: Date.parse(eventTimeAndDateRef.current?.value || ""),
```

Comment:
shouldnt you put the Date.now()? otherwise puttin "" is not good cause it will be Nan